src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -852,7 +846,6 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
     def isEmptyPrefix = (
          isEffectiveRoot                      // has no prefix for real, <empty> or <root>
       || isAnonOrRefinementClass              // has uninteresting <anon> or <refinement> prefix
-      || nme.isReplWrapperName(name)          // has ugly $iw. prefix (doesn't call isInterpreterWrapper due to nesting)
```

Comment:
I guess results in `$read.Foo` and `INSTANCE.Bar`. Hopefully restoring the check with `$read` is kosher and sufficient. I didn't know about these machinations. It makes me wonder if it can be generalized; people want to customize printing of types (simple name, or not; or how about always seeing `collection.Seq` etc).